src/rred_reports/reports/generate.py

```diff
@@ -19,21 +21,34 @@ def __repr__(self) -> str:
         return self.message
 
 
-def generate_report_school(processed_data: pd.DataFrame, template_file: Path, output_dir: Path) -> None:
+def generate_report_school(processed_data: pd.DataFrame, template_file: Path, output_dir: Path, report_year: int) -> None:
     """Generate a report at the school level given a list of school IDs
 
     Args:
         processed_data (pd.DataFrame): Pandas dataframe of processed data
         template_file (Path): The template file to be used
         output_dir (Path): Output directory for saved files
     """
-    school_ids = processed_data.loc[:, "school_id"].unique().tolist()
+    school_ids: list[str] = processed_data.loc[:, "school_id"].sort_values(ascending=True).unique().tolist()
```

Comment:
Do we want the variable annotation here? Personally all for it, but we're not using it with other variables

Reply:
yeah only wanted it there because python thought it returned on object, so then complained when we used it below